spring-boot/autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaAutoConfiguration.java

```diff
@@ -121,6 +128,12 @@ public Server armeriaServer(
 
             server.serviceAt(bean.getPath(), service);
             docServiceRequests.addAll(bean.getExampleRequests());
+            THttpService beanService = bean.getService().as(THttpService.class).get();
```

Comment:
Got it and fixed as your suggestion.

BTW, should `ThriftServiceRegistrationBean.setService` accept `THttpService`, then?

```java
public ThriftServiceRegistrationBean setService(THttpService service) {
    ...
}
````